service/src/test/java/bio/terra/profile/pact/provider/BPMProviderTest.java

```diff
@@ -83,6 +90,16 @@ void verifyPact(PactVerificationContext context) {
     context.verifyInteraction();
   }
 
+  @State("BPM is ok")
+  void getBpmStatus() {
+    when(samService.status()).thenReturn(new SystemStatusSystems().ok(true));
+    // For CloudSQL return value
+    doReturn(jdbcTemplate).when(namedParameterJdbcTemplate).getJdbcTemplate();
```

Comment:
Now I see why I could never get that to work - I didn't realize I needed to proxy that method.